test/hotspot/jtreg/compiler/arguments/TestCompilerCounts.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
IANAL, but shouldn't this include the year?

Reply:
Nope, that's our standard header. Saves us a hassle of updating the years every 365 days ;)